kpops/components/base_components/models/from_section.py

```diff
@@ -36,7 +36,8 @@ def extra_topic_role(cls, values):
 
 
 class FromSection(BaseModel):
-    topics: dict[str, FromTopic]
+    topics: dict[str, FromTopic] = {}
+    components: list[str] = []
```

Comment:
Is it possible to add descriptions here?
Something like
```
topics: dict[str, FromTopic] = Field(
	default={},
	description="Topics to read from.",
)
components: list[str]  = Field(
	default=[],
	description="Components to read from.",
)
```

Reply:
good idea! thanks